File: 20180710/Max Consecutive Ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        result = 0
        count = 0
        for i in nums:
            if i:
                count += 1
            else:
                result = max(result, count)
                count = 0
        result = max(result, count)
        return result  

File: 20180710/Max Consecutive Ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int result = 0;
        int count = 0;
        for(int i = 0; i < nums.length; i ++){
            if(nums[i] == 1){
                count += 1;
            }
            else{
                result = Math.max(result, count);
                count = 0;
            }
        }
        result = Math.max(result, count);
        return result;
    }
}
